ember/app/work-items/controller.js

```diff
@@ -0,0 +1,78 @@
+import Controller from "@ember/controller";
+import { action, set } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import { useCalumaQuery } from "@projectcaluma/ember-core/caluma-query";
+import { allWorkItems } from "@projectcaluma/ember-core/caluma-query/queries";
+
+export default class WorkItemsController extends Controller {
+  queryParams = ["order", "status"];
+
+  // Filters
```

Comment:
I don't think that comment is very helpful, let's remove it

Reply:
Changed